build.sh

```
@@ -2,122 +2,34 @@
 
-#### Remove Ptyxis ####
-# I don't like software redundancy. Kitty will be de default terminal,
-# Ptyxis can be installed as flatpak. When Ptyxis fully support ligatures
-# and font features I'll consider swicthing.
-REMOVE_PACKAGE_LIST="$REMOVE_PACKAGE_LIST ptyxis"
-
-
-#### Actually remove packages ####
-rpm-ostree uninstall $REMOVE_PACKAGE_LIST
-
+rpm-ostree uninstall $(cat $BUILD_FILES_DIR/remove-pkgs)
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)